### ISSUE TITLE
[NuGet.org Bug]: A11y_NuGet Service_Dapper_Release Notes_ScreenReader: On tab navigation to the link under 'Release Notes' tab, NVDA announces the link as the https reference

### ISSUE THREAD

**Issue by advay26** [NuGet/NuGetGallery#9061]
### Impact

It's more difficult to complete my work

### Describe the bug

**Source:** https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1468978

**Actual:**
On tab navigation to the link under 'Release Notes' tab, NVDA announces the link as the https reference
Similar issue is also observed in JAWS
Similar issue is observed in through out the application.

**Expected:**
The name of the link should be descriptive it should have a name that tell its destination point

**User Impact:**
Screen readers users will face difficulty if link is not announced descriptively.

### Repro Steps

1. Launch the application URL: [https://www.nuget.org/packages/Dapper/](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.nuget.org%2f&data=04%7c01%7cv-prajadhav%40microsoft.com%7cf2a4ca1fba7e473d451608d9a59e03f4%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637722920825871875%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=OWWbKuc497S8l6rMh9v0D4GOq3rSkwlcss0RW8CyFkk%3D&reserved=0)
2. "Dapper" page will get open.
3. Tab to 'Release Notes' tab controls and press enter to select it.
4. Run the NVDA
5. Check that screen reader is announcing the link inside 'Release Notes' tab.

### Expected Behavior

The name of the link should be descriptive it should have a name that tell its destination point
